### ISSUE TITLE
UglifyJS setting is breaking external packages relying on currying

### ISSUE THREAD

**Issue by jperasmus** [developit/preact-cli#650]
**Do you want to request a *feature* or report a *bug*?**

Bug - depending on how you want to look at it

**What is the current behaviour?**

Similar to https://github.com/developit/preact-cli/issues/136 the `keep_fargs` setting breaks any external packages that make use of `Function.length`, ie. anything that uses currying like lodash's `curry` method.

**If the current behaviour is a bug, please provide the steps to reproduce.**

Start a new project with this CLI and add lodash's `curry` method. Run production build and the code will be stripped out.

**What is the expected behaviour?**

The code should not be stripped out

**If this is a feature request, what is motivation or use case for changing the behaviour?**

**Please mention other relevant information.**
- Node version: n/a
- npm version: n/a
- Operating system: n/a
- CLI version: v2.2.1
- Browser: n/a

I could get around it for now by specifying a custom config file and popping this in there:

```
export default (config, env, helpers) => {
	const { plugin } = helpers.getPluginsByName(config, 'UglifyJsPlugin')[0] || {};
	if (plugin && plugin.options) {
		plugin.options.compress.keep_fargs = true;
	}
};
```

Not sure how much savings this setting is enabling, but can't think it is much. Consider keeping it as the default `true` instead of `false`.

**Comment by developit** [comment]
Indeed, I doubt this setting is really having that huge of an effect. Should be fine to switch to the default value.
